articles/cosmos-db/how-to-convert-session-token.md

```diff
@@ -0,0 +1,97 @@
+---
+title: How to convert session token formats in .NET SDK - Azure Cosmos DB
+description: Learn how to convert session token formats to ensure compatibilities between different .NET SDK versions
+author: 
+ms.service: cosmos-db
+ms.topic: conceptual
+ms.date: 04/30/2020
+ms.author: vitrinh
+---
+
+# Convert session token formats in .NET SDK
+
+This article explains how to convert between different session token formats to ensure compatibility between SDK versions.
+> [!NOTE]
+> By default, SDK with keeps track of the session token automatically and it will use the most recent session token.  For more information, please visit [Utilize session tokens](https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-manage-consistency#utilize-session-tokens). This article only applies to the following conditions:
```

Comment:
Use relative link to `how-to-manage-consistency.md#utilize-session-tokens`
Some grammar issues.

```suggestion
> By default, the SDK keeps track of the session token automatically and it will use the most recent session token.  For more information, please visit [Utilize session tokens](how-to-manage-consistency.md#utilize-session-tokens). This article only applies to the following conditions:
```